### ISSUE TITLE
Ошибки при старте Vim

### ISSUE THREAD

**Issue by yman45** [Bashka/vim_lib#4]
После обновления Vim при запуске стали сыпаться ошибки. Ситуация одинакова на 3 машинах (ArchLinux, одинаковая версия Vim, примерно идентичный конфиг). Vim в ArchLinux часто обновляется, так что не уверен с какой версии начались ошибки, но предполагаю что с 7.4.1584. Сейчас использую 7.4.1639. На одной из машин снес ~/.vim и вычистил все плагины из .vimrc. Поставил vim_lib - все ок. Поставил vim_plugmanager - получаю ошибки при старте vim. Не уверен к какому плагину относится проблема, но полагаю что все же к vim_lib.

Проверил себя в CentOS на vim 7.4.160 - все работает.

0 ;) yman@ystation ~ $ vim
Error detected while processing /home/yman/.vim/bundle/vim_lib/autoload/vim_lib/sys/NullPlugin.vim:
line   10:
E475: Invalid argument: 4
Error detected while processing /home/yman/.vim/bundle/vim_lib/autoload/vim_lib/sys/System.vim:
line   19:
E475: Invalid argument: 4
line  234:
E475: Invalid argument: 31
Error detected while processing /home/yman/.vim/bundle/vim_lib/autoload/vim_lib/sys/Plugin.vim:
line   79:
E475: Invalid argument: 4
Error detected while processing /home/yman/.vim/bundle/vim_plugmanager/plugin/vim_plugmanager.vim:
line    8:
E118: Too many arguments for function: 4
E15: Invalid expression: s:Plugin.new('vim_plugmanager', '1')
line   13:
E121: Undefined variable: s:p
line   18:
E121: Undefined variable: s:p
line   20:
E121: Undefined variable: s:p
Press ENTER or type command to continue

**Comment by Bashka** [comment]
Да, я уже слышал об этой проблеме в новой версии Vim. Откат к более старой (на пример 7.4.1190) решает эту проблему. Не успел еще затестить у себя.

**Comment by palandovalex** [comment]
Добрый день, товарищ Bashka. Хотел бы уточнить - ваша самодельная vim инфраструктура поддерживается, или проект совсем заглох? Насколько ваш проект с совместим с NeoVim?
